Sources/App/Settings/Eureka/RealmSection.swift

```diff
@@ -93,7 +93,7 @@ public final class RealmSection<ObjectType: Object>: Section {
             }.flatMap { $0 }
 
             // note: eureka has a bug if we don't wrap this in Array where it duplicates
```

Comment:
Did you test this?

Reply:
@RomanPodymov I see your reaction to Zac's comment, could you confirm you tested the part of the code you have updated? I understand it looks so simple that it is not required but sometimes we break things with small changes like this, and I really appreciate the "scout" on HA code, thank you, so please let me know otherwise I will close this PR for now.

Reply:
Hello @bgoncal 
Let's close it for now.